include/CppUTest/UtestMacros.h

```diff
@@ -200,6 +200,29 @@
 #define UNSIGNED_LONGS_EQUAL_LOCATION(expected, actual, text, file, line)\
   { UtestShell::getCurrent()->assertUnsignedLongsEqual((unsigned long)expected, (unsigned long)actual, text, file, line); }
 
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
I don't know about cmake, but... when you install the apt-get package or the homebrew package, at the moment the config.h that is generated by autoconf is NOT installed yet. This has been something to-do as it would require renaming the config.h to something else (as the filename config.h is a bit too easy to lead to mistakes) and then prefix the configure results and install it.

That means that currently the system environment #defines discovered by autotools aren't kept after it is installed. Thus, it could be very common for someone to install CppUTest and not have USE_LONG_LONG #defined. The question is whether we want to let them #define it in their project or try to get it to "just work". By removing the #ifdef, we might be able to get a run-time (assigning an int to a struct, a linker error (not defined the StringFrom methods), or a run-time error (unsupported feature))

My guess is that this would be better, as in most cases such a thing shouldn't happen.